### DISCUSSION TITLE
Documentation Feedback

### DISCUSSION THREAD

**Discussion by BobbieGoede** [nuxt-modules/i18n#3171] [General]
We recognize the documentation can be improved in both structure and material, if you have suggestions or general feedback on changes we could make leave them here!

Some changes I am already working on (v9 docs):

- [x] Merging the options pages into a single page
- [x] Move dynamic parameters section to custom paths page
- [x] Separating composables into individual pages
- [x] Separating component into individual pages

If you know of other package documentation that we could look at for reference, let us know which and what they do well 🙏

**Comment by darthf1** [comment]
~~I was just looking through the `v9` documentation, and I could not find anything about the `experimental` option of the nuxt config?~~

Nm, its here: https://i18n.nuxtjs.org/docs/v9/api/options#experimental
